AutomationApp/LoggerCallbackImpl.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+
+namespace AutomationApp
+{
+    class LoggerCallbackImpl : ILoggerCallback
+    {
+        private readonly StringBuilder _logCollector = new StringBuilder();
+
+        public void Log(Logger.LogLevel level, string message, bool containsPii)
```

Comment:
The LogLevel is an enum inside of the Logger class, which is why I need to use the qualified name. If there is a different way to reference it, let me know.